keras_cv/version_check.py

```diff
@@ -18,7 +18,7 @@
 import tensorflow as tf
 from packaging.version import parse
 
-MIN_VERSION = "2.12.0"
+MIN_VERSION = "2.11.0"
```

Comment:
Note: I verified that TF 2.11 is compatible with the other parts of the package.

This got switched to 2.12 during the Keras Core rebase, and I think it was incidental as I don't see any reasoning for it in the diff.